tests/roles/dataplane_adoption/tasks/nova_ffu.yaml

```diff
@@ -65,28 +65,14 @@
     apiVersion: v1
```

Comment:
Note:  The changes in this file should not be included.  They were meant for testing only.

The changes that are needed here would be provided as part of https://github.com/openstack-k8s-operators/data-plane-adoption/pull/376 - which should be a depends-on for this patch.